pkg/vm/stack_item.go

```diff
@@ -167,6 +175,11 @@ func (i *BigIntegerItem) Bytes() []byte {
 	return emit.IntToBytes(i.value)
 }
 
+// TryBytes implements StackItem interface.
+func (i *BigIntegerItem) TryBytes() ([]byte, error) {
+	return i.Bytes(), nil
```

Comment:
Shouldn't it be `emit.IntToBytes(i)`? I think there should be some symmetry between this and `(*Element).BigInt()`.

Reply:
Well, `Bytes()` is exactly `emit.IntToBytes()`. This way there is less code duplication.

Reply:
Ah, sorry, confused `i.Bytes()` to `i.value.Bytes()` for some reason.